README.md

```diff
@@ -41,6 +41,23 @@ $ aws --region $AWS_DEFAULT_REGION cloudformation create-stack --stack-name hybo
 
 You can also deploy branches of the hybox application repository by setting the `WebappS3Key` parameter for your stack to point at the branch-specific deployment artifact (e.g. `hyku/branch/branch-name/hyku.zip`)
 
+The stack will spin up in the following order:
+
+```console
+|- stack
```

Comment:
Solr and codepipeline added. Travis is only used for CI and doesn't come up as part of the regular deployment stack.